wire-ios-request-strategy/Sources/Request Strategies/User/UserProfileRequestStrategy.swift

```diff
@@ -58,12 +62,11 @@ public class UserProfileRequestStrategy: AbstractRequestStrategy, IdentifierObje
 
     public override func nextRequestIfAllowed(for apiVersion: APIVersion) -> ZMTransportRequest? {
         fetchAllConnectedUsers(for: apiVersion)
-        switch apiVersion {
-        case .v0:
-            return userProfileByID.nextRequest(for: apiVersion)
-        case .v1, .v2, .v3, .v4, .v5:
-            return userProfileByQualifiedID.nextRequest(for: apiVersion)
-        }
+
+        return
+        userProfileByID.nextRequest(for: apiVersion) ??
+        userProfileByQualifiedID.nextRequest(for: apiVersion) ??
+        actionSync.nextRequest(for: apiVersion)
```

Comment:
I think you can write this as

```swift
return [userProfileByID, userProfileByQualifiedID, actionSync].nextRequest(for: apiVersion)
```

Reply:
Yep, we can write it like this. I fixed it.